### PR TITLE
Enable to draw the curve eval only for a specific hue curve.

### PR DESCRIPTION
**Pull request by larochj**

### CODE REVIEW

**Review by doug-walker — CHANGES_REQUESTED** [review]
This looks better, thanks.

File: src/OpenColorIO/ops/gradinghuecurve/GradingHueCurveOpGPU.cpp [hunk 1]
```diff
@@ -205,6 +224,11 @@ void AddGCPropertiesUniforms(GpuShaderCreatorRcPtr & shaderCreator,
                DynamicPropertyGradingHueCurveImpl::GetMaxCoefs(),
                propNames.m_coefs);
     AddUniform(shaderCreator, getLB, propNames.m_localBypass);
+
+    AddSingleIntUniform(shaderCreator, 
+                        getNCurveToDraw, 
+                        getCurveToDraw, 
+                        propNames.m_curveToDraw);
```
[the remark this answers is not in the document]
Comment (larochj) [reply src/OpenColorIO/ops/gradinghuecurve/GradingHueCurveOpGPU.cpp]: This code is from before I removed the Int, it is no longer in the PR

File: include/OpenColorIO/OpenColorTransforms.h [hunk 2]
```diff
@@ -844,6 +844,9 @@ class OCIOEXPORT DynamicPropertyGradingHueCurve
     /// Will throw if value is not valid.
     virtual void setValue(const ConstGradingHueCurveRcPtr & value) = 0;
 
+    virtual const int* getCurveToDraw() const = 0;
+    virtual void setCurveToDraw( HueCurveType curveToDraw ) = 0;
```
[the remark this answers is not in the document]
Comment (larochj) [reply include/OpenColorIO/OpenColorTransforms.h]: This code is from before I removed the Int, it is no longer in the PR

File: src/OpenColorIO/ops/gradinghuecurve/GradingHueCurveOpGPU.cpp [hunk 3]
```diff
@@ -478,7 +510,7 @@ void GetHueCurveGPUShaderProgram(GpuShaderCreatorRcPtr & shaderCreator,
     switch (dir)
```
[the remark this answers is not in the document]
Comment (larochj) [reply src/OpenColorIO/ops/gradinghuecurve/GradingHueCurveOpGPU.cpp:486]: That's right, fixed

File: include/OpenColorIO/OpenColorTransforms.h [hunk 4]
```diff
@@ -1308,6 +1311,11 @@ class OCIOEXPORT GradingHueCurveTransform : public Transform
     virtual bool getBypassLinToLog() const = 0;
     virtual void setBypassLinToLog(bool bypass) = 0;
```
[the remark this answers is not in the document]
Comment (larochj) [reply include/OpenColorIO/OpenColorTransforms.h:1310]: Fixed

File: src/OpenColorIO/ops/gradinghuecurve/GradingHueCurveOpData.cpp [hunk 5]
```diff
@@ -228,6 +231,7 @@ bool GradingHueCurveOpData::equals(const OpData & other) const
     if (m_direction      != rop->m_direction ||
         m_style          != rop->m_style ||
         m_bypassLinToLog != rop->m_bypassLinToLog ||
+        m_drawCurveOnly != rop->m_drawCurveOnly ||
```
[the remark this answers is not in the document]
Comment (larochj) [reply src/OpenColorIO/ops/gradinghuecurve/GradingHueCurveOpData.cpp]: Fixed, I aligned the != with the rest above

**Review by doug-walker — APPROVED** [review]
Looks good!

Sorry about some of the comments, they were left over from an earlier review and I didn't realize they got added to my review of your second pass.

**Pull request merged**